common/cultures/rathnir_cultures.txt

```diff
@@ -281,6 +281,26 @@ vastallosi_group = {
 	}
 }
 
+ardatian_group = {

```

Comment:
names are not alphabetised, you can use something like this: https://alphabetizer.flap.tv/

Reply:
names do not need to be alphabetized for code